lib/carin_for_blue_button_test_kit/generated/v1.1.0/explanation_of_benefit_inpatient_institutional/explanation_of_benefit_inpatient_institutional_explanation_of_benefit_all_search_test.rb

```diff
@@ -14,7 +14,9 @@ class ExplanationOfBenefitInpatientInstitutionalExplanationOfBenefit_AllSearchTe
       def self.properties
         @properties ||= SearchTestProperties.new(
           resource_type: 'ExplanationOfBenefit',
-        search_param_names: ['_include']
+        search_param_names: ['_include'],
+        include_paths: ["patient", "provider", "careTeam.provider", "insurance.coverage", "insurer"],
+        include_targets: ["Patient", "Organization", "Practitioner", "PractitionerRole", "Coverage"]
```

Comment:
This is not correct. For example, path `provider` has three targets `['Practitioner', 'Organization', 'PractitionerRole']`, but this structure does not reflect that. 

I propose the input like this:
```
search_param_names: ['_include'],
include_parameters: [
  {
    path: 'patient',
    target: ['Patient']
  },
  {
    path: 'provider',
    target: ['Organization', 'Practitione', 'PractitionerRole']
  },
  {
    path: 'careTeam.provider',
    target: ['Organization', 'Practitione', 'PractitionerRole']
  },
  {
    path: 'insurance.coverage',
    target: ['Coverage']
  },
  {
    path: 'insurer',
    target: ['Organization']
  }
]
```

Reply:
Made this change and updated the include test to work with this new structure